test-utils/src/bc5SupportDev/java/com/revenuecat/purchases/utils/stubsBillingClient5.kt

```diff
@@ -0,0 +1,73 @@
+package com.revenuecat.purchases.utils
+
+import com.android.billingclient.api.BillingClient
+import com.android.billingclient.api.Purchase
+import com.android.billingclient.api.PurchaseHistoryRecord
+import com.android.billingclient.api.SkuDetails
+import org.json.JSONArray
+
+@SuppressWarnings("LongParameterList", "MagicNumber")
+fun stubSkuDetails(
```

Comment:
these are the same for now but will change when we do the SkuDetails -> ProductDetails migration next